source/library/CabalGild/Unstable/Type/TestedWith.hs

```diff
@@ -1,13 +1,11 @@
 module CabalGild.Unstable.Type.TestedWith where
 
-import qualified CabalGild.Unstable.Type.VersionRange as VersionRange
 import qualified Data.Ord as Ord
 import qualified Distribution.FieldGrammar.Newtypes as Newtypes
 import qualified Distribution.Parsec as Parsec
 import qualified Distribution.Pretty as Pretty
 
--- | This type exists to provide an 'Ord' instance for 'Newtypes.TestedWith',
--- which was added in @Cabal-syntax-3.10.1.0@.
+-- | This type exists to provide an 'Ord' instance for 'Newtypes.TestedWith'.
```

Comment:
This type does not have an `Ord` instance at all: https://hackage.haskell.org/package/Cabal-syntax-3.14.1.0/docs/Distribution-FieldGrammar-Newtypes.html#t:TestedWith